src/v2/BaseLoginRouter.js

```diff
@@ -153,7 +153,25 @@ export default Router.extend({
   },
 
   handleIdxResponseFailure (error = {}) {
-    // 1. loosely check whether is IDX error response
+    // special case: `useInteractionCodeFlow` is true but the Org is not using OIE
+    const featureMsg = 'The requested feature is not enabled in this environment.';
+    if (error?.error === 'access_denied' && error.error_description === featureMsg) {
```

Comment:
will message always be English even on different locale (backend)

Reply:
This can be problematic to check the english text. Its defined as an [enum](https://github.com/okta/okta-core/blob/master/components/core/tokens/api/src/main/java/com/okta/monolith/platform/tokens/dto/OAuth2ErrorCode.java#L82) in the monolith. The [description is constructed using a locale in context](https://github.com/okta/okta-core/blob/master/components/core/tokens/api/src/main/java/com/okta/monolith/platform/tokens/dto/OAuth2ErrorCode.java#L338) which may result in a non english text.

Reply:
@aarongranick-okta you mentioned over slack not to use this as a check right ?

Reply:
We can remove the check on description, but then I think we'd want to change the error message to reflect the loss of certainty related to "feature not enabled". It would now be an "access denied" message.

Reply:
We can check for `error.feature.disabled` key instead.. ?